src/services/calculatorService.js

```diff
@@ -0,0 +1,84 @@
+import { state, setOutput, setState } from '../store/calculatorStore.js';
+import { isEmpty, isZero, isEqualOperation } from '../utils/helpers.js';
+
+const PERCENT = 0.01;
+const REVERS = 2;
+const digits = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '.'];
+const actions = ['-', '+', '*', '/', '+/-', '%'];
+const clearArg = {
```

Comment:
```suggestion
const defaultState = {
```

И перенеси её в `store`
и там сделай 
`const state = defaultState;`
А потом меняй именно state уже, как обычно